java-surefire/src/main/java/org/sonar/plugins/surefire/SurefireExtensions.java

```diff
@@ -34,9 +34,23 @@ private SurefireExtensions() {
 
   public static List getExtensions() {
     return ImmutableList.of(
+        /**
+         * @deprecated
+         * @since 4.11
+         */
         PropertyDefinition.builder(SurefireUtils.SUREFIRE_REPORTS_PATH_PROPERTY)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "SUREFIRE_REPORTS_PATH_PROPERTY"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)